src/HdbConfigurationManager.cpp

```diff
@@ -326,37 +318,23 @@ void HdbConfigurationManager::init_device()
 	*attr_SetTTL_read = 0;
 	*attr_SetStrategy_read = CORBA::string_dup("");
 	archiver_t tmp;
-	for(vector<string>::iterator it = archiverList.begin(); it!= archiverList.end(); it++)
+	for(auto it = archiverList.begin(); it!= archiverList.end(); it++)
```

Comment:
Since we use auto and C++11 features, we could be using something like:
for (const auto & archname : archiverList)
{
 ...
     tmp.dp = std::unique_ptr<Tango::DeviceProxy>(new Tango::DeviceProxy(archname));
}

This comment applies to the other loops in this file.

Reply:
My original goal was to remove as much compilations warnings as possible, but the amount of work to be done, especially in the es broke my spirit.
I could finish the work on the cm though, it's smaller…